ios/Runner/AppDelegate.swift

```diff
@@ -1,13 +1,27 @@
 import UIKit
 import Flutter
+import workmanager
 
 @UIApplicationMain
 @objc class AppDelegate: FlutterAppDelegate {
   override func application(
     _ application: UIApplication,
     didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?
   ) -> Bool {
+    if #available(iOS 10.0, *) {
+      UNUserNotificationCenter.current().delegate = self as? UNUserNotificationCenterDelegate
+    }
     GeneratedPluginRegistrant.register(with: self)
+    WorkmanagerPlugin.registerTask(withIdentifier: "com.cBreez.paymentreceived")
+    UIApplication.shared.setMinimumBackgroundFetchInterval(TimeInterval(60*15))
     return super.application(application, didFinishLaunchingWithOptions: launchOptions)
   }
+
+  override func applicationDidBecomeActive(_ application: UIApplication) {
+    signal(SIGPIPE, SIG_IGN);
```

Comment:
Can you please explain this?

Reply:
It's to prevent app from freezing/crash when it's brought to foreground again.
https://github.com/flutter/flutter/issues/47203#issuecomment-590834018